Ports/bash/patches/0002-Remove-getconf-from-things-to-build.patch

```diff
@@ -0,0 +1,1355 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Fabian Dellwing <fabian.dellwing@gmail.com>
+Date: Mon, 24 Apr 2023 16:00:48 +0200
+Subject: [PATCH] Remove getconf from things to build
+
+---
+ examples/loadables/Makefile.in |    6 +-
+ examples/loadables/getconf.c   | 1163 --------------------------------
+ examples/loadables/getconf.h   |  136 ----
+ 3 files changed, 1 insertion(+), 1304 deletions(-)
+ delete mode 100644 examples/loadables/getconf.c
+ delete mode 100644 examples/loadables/getconf.h
+
+diff --git a/examples/loadables/Makefile.in b/examples/loadables/Makefile.in
+index 956f018933f861ff157841f42247a778f04ba909..1cef689e1e0c15d7986d54f1f9e67e18103ae3b5 100644
+--- a/examples/loadables/Makefile.in
++++ b/examples/loadables/Makefile.in
+@@ -104,7 +104,7 @@ INC = -I. -I.. -I$(topdir) -I$(topdir)/lib -I$(topdir)/builtins -I${srcdir} \
+ ALLPROG = print truefalse sleep finfo logname basename dirname fdflags \
+ 	  tty pathchk tee head mkdir rmdir mkfifo mktemp printenv id whoami \
+ 	  uname sync push ln unlink realpath strftime mypid setpgid seq rm \
+-	  accept csv dsv cut stat getconf
++	  accept csv dsv cut stat
+ OTHERPROG = necho hello cat pushd asort
+ 
+ all:	$(SHOBJ_STATUS)
```

Comment:
We can probably strip down the patch to this chunk alone, which should keep future conflicts minimal.